Tests/Unit/EmogrifierTest.php

```diff
@@ -1997,10 +1997,7 @@ public function emogrifierMapsSuitableCssToHtmlIfFeatureIsEnabled($body, $css, $
         $this->subject->enableCssToHtmlMapping();
         $html = $this->subject->emogrify();
 
-        self::assertContains(
-            '<' . $tagName . ' ' . $attributes,
-            $html
-        );
+        self::assertRegExp('/<' . preg_quote($tagName) . '[^>]+' . preg_quote($attributes) . '/', $html);
```

Comment:
You'll also need to provide `'/'` as the second parameter to `preg_quote` so that the delimiter also will get escaped.